README.md

```diff
@@ -1,66 +1,228 @@
+
+### Docker Setup
+The light weight containerized version of the HPE OneView SDK for Terraform is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hewlettpackardenterprise/hpe-oneview-sdk-for-terraform). The Docker Store image tag consist of two sections: <sdk_version-OV_version>
 
 ```bash
-# Download the source code for terraform-provider-oneview
-# and build the needed binary, by saving it inside $GOPATH/bin
-$ go get -u github.com/HewlettPackard/terraform-provider-oneview
+# Download and store a local copy of oneview-sdk-for-terraform and use it as a Docker Image.
+$ docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.7.0-12-OV5.6
+# Run docker commands below given, which  will in turn create a sh session 
+# where you can create files, issue commands and execute the examples.
+$ docker run -it docker pull hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:v1.7.0-12-OV5.6 /bin/sh
+```
 
-# Copy the binary to have it along the terraform binary
-$ mv $GOPATH/bin/terraform-provider-oneview /usr/local/bin/terraform
+### Local Setup
+
+Local installation requires
+- Installing Go
+```bash 
+$ apt-get install build-essential git wget
+$ wget https://golang.org/dl/go1.15.7.linux-amd64.tar.gz
+
+#unzip and untar the file 
+$ tar -zxvf go1.15.7.linux-amd64.tar.gz
+
+# move it to /usr/local/ and create directory for Go.
+$ mv go/ /usr/local/ 
+$ mkdir ~/go
+```
+```bash 
+# Setting Environment Variable 
+$ export GOROOT=/usr/local/go
+$ export GOPATH=$HOME/go 
+$ export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 ```
 
+- Installing Terraform
 
+	Install Terraform 0.12.x [from here](https://www.terraform.io/downloads.html) and save it into `/usr/local/bin/terraform` folder (create it if it doesn't exists).
 
-## Using `terraform-provider-oneview` with Docker
+```bash 
+```
 
+- Install Oneview Terraform Provider SDK
+```go
+# Download the source code for terraform-provider-oneview
+# Build the needed binary
+# Get the branch 'Terraform-0.12' terraform-provider-oneview which is supports Terraform v0.12.x.
 
-We also provide a lightweight and easy way to test and execute `terraform-provider-oneview`. The `hewlettpackardenterprise/hpe-oneview-sdk-for-terraform:<tag>` docker image contains an installation of Terraform and our provider you can use by just pulling down the Docker Image:
-Docker Store image tag consist of two sections: <sdk_version-OV_version>
+$ git clone -b Terraform-0.12  https://github.com/HewlettPackard/terraform-provider-oneview.git
+$ cd terraform-provider-oneview
+# Build the provider
+$ go build -o terraform-provider-oneview
+# Create the plugin location if it does not exist and copy the  provider binary there.
+$ mkdir -p ~/.terraform.d/plugins/
+$ mv terraform-provder ~/.terraform.d./plugins/
```

Comment:
Please make line 83 to below, 
$  mv terraform-provider-oneview ~/.terraform.d/plugins/
I verified this, works for me.

Reply:
Done